### ISSUE TITLE
repository.treeBuilder() throws 'Error: Repository repo is required.'

### ISSUE THREAD

**Issue by petemill** [nodegit/nodegit#813]
Seems like it's impossible to create a treeBuilder without a source tree (even through source is supposed to be optional), e.g. to create initial commit.

repository instance function:
```javascript
repo.treeBuilder()
```
```
Error: Repository repo is required.
```

TreeBuilder.create:
```javascript
Git.Treebuilder.create(repo)
```
```
Error: Callback is required and must be a Function.
```

**Comment by petemill** [comment]
Solved by #819 but need to call `Git.Treebuilder.create(repo, null)`

**Comment by johnhaley81** [comment]
#819 is merged so this should be good now.
